pkg/api/const.go

```diff
@@ -72,6 +76,8 @@ const (
 	// DefaultKubernetesFirstConsecutiveStaticIPOffsetVMSS specifies the IP address offset of master 0 in VMSS
 	// when VNET integration is enabled.
 	DefaultKubernetesFirstConsecutiveStaticIPOffsetVMSS = 4
+	// DefaultSubnetNameResourceSegmentIndex specifies the default subnet name resource segment index.
+	DefaultSubnetNameResourceSegmentIndex = 10
```

Comment:
where is this used?

Reply:
https://github.com/tariq1890/acs-engine/blob/extract_output_params/pkg/api/types.go#L784

Reply:
ah my search failed me